config/api.yml

```diff
@@ -0,0 +1,24 @@
+:collections:
+  :red_hat_cloud_service_providers:
+    :description: Red Hat Cloud Service Providers
+    :identifier: redhat_cloud_service
```

Comment:
Is the product feature redhat vs red_hat?  If so, we should update the product feature.